proto/firecracker.proto

```diff
@@ -36,6 +36,7 @@ message CreateVMRequest {
 
 message StopVMRequest {
     string VMID = 1;
+    uint32 VMShutDownTimeOut = 2;
```

Comment:
For this variable, units matter, so can we name it more like `VMShutdownTimeoutSeconds`?